modules/extensions-xml-support/src/test/resources/module-custom-types/module-custom-types-catalog.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file has the same content as module-param-custom-types-catalog.xml. Can you use just one of them?